### PR TITLE
feat: added strings for translations for lookup field

### PR DESCRIPTION
**Pull request by gosiexon-zen**
## Description

We are introducingLookup fields for custom objects in the Copenhagen Theme v4 in new request form and we need to add two short strings for translation.

[Jira epic](https://zendesk.atlassian.net/jira/software/c/projects/GG/boards/1258?selectedIssue=GG-3522)
[Figma design](https://www.figma.com/design/y9oQEXp0WSp1HAAqRcrhF3/End-User-Lookup-Fields-2024-Q1?node-id=695-37365&t=BXTZ6vkX87j1qudn-0)

## Screenshots
Screenshot 2024-08-07 at 16 53 52
Screenshot 2024-08-07 at 16 53 30

## Checklist

- [ ] :green_book: all commit messages follow the [conventional commits](https://conventionalcommits.org/) standard
- [ ] :arrow_left: changes are compatible with RTL direction
- [ ] :wheelchair: Changes to the UI are [tested for accessibility](./../README.md#accessibility-testing) and compliant with [WCAG 2.1](https://www.w3.org/TR/WCAG21/).
- [ ] :memo: changes are tested in Chrome, Firefox, Safari and Edge
- [ ] :iphone: changes are responsive and tested in mobile
- [ ] :+1: PR is approved by @zendesk/vikings

### CODE REVIEW

File: src/modules/new-request-form/translations/en-us.yml [hunk 1]
```diff
@@ -156,3 +156,13 @@ parts:
       title: "Additional label shown above the credit card field, informing the user that only the last 4 digits are required."
       screenshot: "https://drive.google.com/file/d/13Pz2p5q7FHX9FKT6MUR6Sg5vWy55X0kP/view?usp=drive_link"
       value: "(Last 4 digits)"
+  - translation:
+      key: "new-request-form.lookupfield.loading-options"
```
Comment (Fredx87) [inline src/modules/new-request-form/translations/en-us.yml:160]: I would change the key to make it more readable
```suggestion
      key: "new-request-form.lookup-field.loading-options"
```

File: src/modules/new-request-form/translations/en-us.yml [hunk 2]
```diff
@@ -156,3 +156,13 @@ parts:
...
+      title: "Message shown in combobox when searched options are loading"
+      screenshot: "https://drive.google.com/file/d/1_TeqU5hihEJb03q6adVI0-nuklYask1d/view?usp=drive_link"
+      value: "Loading items..."
+  - translation:
+      key: "new-request-form.lookupfield.no-matches.found"
```
Comment (Fredx87) [inline src/modules/new-request-form/translations/en-us.yml:165]: Same here
```suggestion
      key: "new-request-form.lookup-field.no-matches-found"
```

**Review by Fredx87 — CHANGES_REQUESTED** [review]
Looks good, I would just change the keys a bit

File: src/modules/new-request-form/translations/en-us.yml (hunk 1 above)
Comment (gosiexon-zen) [reply src/modules/new-request-form/translations/en-us.yml]: ok, changed

File: src/modules/new-request-form/translations/en-us.yml (hunk 2 above)
Comment (gosiexon-zen) [reply src/modules/new-request-form/translations/en-us.yml]: changed

**Review by Fredx87 — APPROVED** [review]

**Review by kubraokcu — APPROVED** [review]
👍🏼🌐

**Pull request merged**